force-app/main/default/classes/CaseRoutingConfigurationTest.cls

```diff
@@ -0,0 +1,20 @@
+@IsTest
```

Comment:
When writing test methods, good practice is to user the Test.startTest() - Test.stopTest() clause, even when you don't need to reset limits.
Using them it is easier to see which part is "preperation part" and which one is actual test core.